src/components/LocaleLink.js

```diff
@@ -21,8 +21,8 @@ const LocaleLink = ({ children, to, locale, ...other }) =>
       }
 
       localeTo = localeTo.replace(
-        new RegExp(`^/${supportedLocales.join('|')}(/.*)$`),
-        `${locale === defaultLocale ? '' : '/' + locale}$1`
+        new RegExp(`^/${supportedLocales.join('/|')}(.*)$`),
```

Comment:
Should we also prefix with `/` so it's very clearly `/<locale>/`?

Reply:
In particular wondering if this construction still results in weirdness e.g. for `/news/2089-04-12-cathodes-and-anode/`  and gets us `-an` at the end instead.

Reply:
Ah yes, good call! Will adjust.

Reply:
Addressed!